File: phaseGate1/java/Day2/TaskTwoTest.java
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class TaskTwoTest {
	@Test
	public void testGradeScore() {
	int actual = TaskTwo.gradeScore(10,10,10);
	int expected = 10;
	assertEquals(actual,expected);	

	}

}

File: phaseGate1/python/Day2/TaskTwoTest.py
from unittest import TestCase
import TaskTwo

class TestGradeScore(TestCase):
	def test_grade_score_of_two_number(self):
		element1 = 10
		element2 = 10
		element3 = 10
		actual = TaskTwo.gradeScore(element1,element2, element3)
		expected = 10
		self.assertEqual(actual, expected)